src/js/jsx/IconBar.jsx

```diff
@@ -74,12 +84,12 @@ define(function (require, exports, module) {
 
         componentDidMount: function () {
             this._updatePanelSizesDebounced = synchronization.debounce(this._updatePanelSizes, this, 500);
```

Comment:
an alternate suggestion is to just stuff this inside an anon function right here, instead of creating `_handleDisplayConfigurationChanged`